media_test_harness/testcase.py

```diff
@@ -37,24 +38,24 @@ def log_video_debug_lines(self):
             if debug_lines:
                 self.marionette.log('\n'.join(debug_lines))
 
-    def run_playback(self, interval=1, set_duration=None,
-                     stall_wait_time=10):
+    def run_playback(self, video):
         with self.marionette.using_context('content'):
-            for url in self.video_urls:
-                self.logger.info(url)
-                video = VP(self.marionette, url, interval=interval,
-                           set_duration=set_duration,
-                           stall_wait_time=stall_wait_time)
-                verbose_until(Wait(video, timeout=30), video,
-                              playback_started)
+            self.logger.info(video.test_url)
+            try:
+                verbose_until(Wait(video, interval=video.interval,
+                                   timeout=video.expected_duration * 1.3 +
```

Comment:
Multiplying the expected duration by 1.3 still bothers me. Adding stall_wait_time should be good enough. Do we have evidence that a video is exceeding the expected_duration + stall_wait_time threshold, and it is not a bug?